### DISCUSSION TITLE
Mattermost penpot/mattermost-omnibus

### DISCUSSION THREAD

**Discussion by jemedeiros** [uxbox/uxbox#2530] [Q&A]
I have a probably dumb question: 

Is it correct to understand that the Mattermost code at /penpot/mattermost-omnibus means that the penpot instance should have a chat running too in the background ? Or that I should be able to add it to the instance like a plugin ?
I cannot find a chat function in the running instance; nor am I sure how to add the mattermost chat to it.

Thank you for any clarification.

**Comment by niwinz** [comment]
The mattermost omnibus was a try to integrate penpot on the mattermost
install process, but it was discontinued before initial release of the same.

So this code right now is just a legacy that does not work and we do not
expect make it working.
